site/src/pages/UserSettingsPage/WorkspaceProxyPage/WorkspaceProxyRow.tsx

```diff
@@ -26,12 +26,30 @@ export const ProxyRow: FC<ProxyRowProps> = ({ proxy, latency }) => {
 	// All users can see healthy/unhealthy, some can see more.
 	let statusBadge = <ProxyStatus proxy={proxy} />;
 	let shouldShowMessages = false;
+	const extraWarnings: string[] = [];
+	if (latency?.nextHopProtocol) {
+		switch (latency.nextHopProtocol) {
+			case "http/0.9":
+			case "http/1.0":
+			case "http/1.1":
+				extraWarnings.push(
+					// biome-ignore lint/style/useTemplate: easier to read short lines
+					`Requests to the proxy are using ${latency.nextHopProtocol}, ` +
+						"the server might not support HTTP/2. " +
+						"For usability reasons, HTTP/2 or above is recommended. " +
+						"Pages may fail to load if the web browser's concurrent " +
+						"connection limit is reached.",
```

Comment:
I find this description really unclear. Requests to the proxy from where? What server? Why is there a comma separating what feels like two sentences? I think this needs a bit more refinement

Reply:
How about:

```
`Requests to the proxy from current browser are using "${latency.nextHopProtocol}". ` +
						"The proxy server might not support HTTP/2. " +
						"For usability reasons, HTTP/2 or above is recommended. " +
						"Pages may fail to load if the web browser's concurrent " +
						"connection limit per host is reached.",
```